CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+- **BREAKING**: The `InstallAppPayload` now unifies all settings that are per role in a `role_settings` field and as part of this change adds the option to specify custom modifiers at install time to override the modifiers defined in the dna manifest(s).
```

Comment:
The release automation should update the root changelog. Changes should be added to the `crates/holochain/CHANGELOG.md`. 

This change wasn't on the PR to be backported, so it shouldn't have come in with a `git cherry-pick`. Is that the process you're using for backports or something else?

Reply:
Oh okay, I didn't know that. No I actually did the backport manually (not git cherry-pick) because I wanted to include some of the changes that Jost proposed after it had already been merged too. I removed that part again now.